resources/views/backend/worldmap/index.blade.php

```diff
@@ -38,7 +38,106 @@
                 </div>
                 <div class="card shadow mb-2">
                     <div class="card-body">
-                        <div id="map"></div>
+                        <div class="row">
+                            <div class="col-12 col-sm-6 col-md-5 col-lg-4 col-xl-4">
+                                <div class="u-vmenu">
+                                    <ul>
+                                        <li> <a href="#">Towns</a>
```

Comment:
All that Textes, dunno If I would add a translation for it or let it be, because ingame is the same text.

Reply:
Translation would be the best but sounds like a big section will be required for silkroad texts.

Reply:
I think we can make it first. Translating the Silkroad texts of the map can still be done.